ACadSharp/Entities/MultiLeader.cs

```diff
@@ -345,7 +346,7 @@ public override CadObject Clone()
 
 			clone.ContextData = (MultiLeaderAnnotContext)this.ContextData?.Clone();
 
-			foreach (var att in BlockAttributes)
+			foreach (var att in BlockAttributes.ToList())
```

Comment:
Is there a particular case to add the `ToList()`?

Reply:
@nka1994, why did you add `ToList()`?